protocol/database-scripting/README.md

```diff
@@ -0,0 +1,102 @@
+Database Scripting
+==============
+
+A protocol for running console scripts on the Healthify app's production database.
+
+Development
+-----------
+
+Develop the scripting story similarly to any other story. In this case, this would
+entail running your script at the rails console of your local server.
+
+While some scripts only lend themselves to being performed via pure SQL, where it
+is possible, you should leverage ActiveRecord to perform our INSERT/UPDATE/DELETE
+operations so that we leverage ActiveRecord validations and callbacks.
+
+While developing the script, it is critical to plan our remediation strategy if
+something does go awry.
```

Comment:
I wasn't thinking of anything more complex than peppering the script with useful `puts` calls, so that after it's finished it's clear what has happened.